### PR TITLE
Implemented Auth0

### PR DESCRIPTION
**Pull request by Nandan-unni-zz**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
+  PrivateRoute,
+  PublicRoute,
+  routes,
+} from "./presentation/common/Routes";
 
 // Component imports
 import Home from "./presentation/Home";
+import Login from "./presentation/Login";
+import Initial from "./presentation/Initial";
+import Auth0 from "./presentation/Auth0";
```
Comment (AbhinavRajesh) [inline src/App.tsx]: Should we make it to one line import? Like,

```
import { Home, Login, Initial } from "...";
```

File: src/common/config/index.tsx [hunk 2]
```diff
@@ -5,38 +5,48 @@
...
-const production = {
-  ...development,
+const development = {
+  ...local,
+  CLIENT_URL: "https://chatbotish-dash-dev.vercel.app",
 };
 
 const staging = {
   ...development,
+  CLIENT_URL: "https://chatbotish-dash.vercel.app",
```
Comment (AbhinavRajesh) [inline src/common/config/index.tsx]: Is this the correct staging `CLIENT_URL`?

**Review by AbhinavRajesh — APPROVED** [review]
Check the comments and then it's good to merge! :) @Nandan-unni

File: src/common/config/index.tsx [hunk 3]
```diff
@@ -5,38 +5,48 @@
...
+  CLIENT_URL: "http://localhost:3000",
+  AUTH0_DOMAIN: "dev-qlg7c9vz.us.auth0.com",
+  AUTH0_CLIENT_ID: "czQVb2eCU05djAnNy96MAh5Ki9JxKY2e",
 };
 
-const production = {
-  ...development,
+const development = {
+  ...local,
+  CLIENT_URL: "https://chatbotish-dash-dev.vercel.app",
```
Comment (AbhinavRajesh) [inline src/common/config/index.tsx]: Is this the correct link for dev?

**Review by AbhinavRajesh — COMMENTED** [review]
Dev client url

File: src/App.tsx (hunk 1 above)
Comment (Nandan-unni-zz) [reply src/App.tsx]: done

File: src/common/config/index.tsx (hunk 2 above)
Comment (Nandan-unni-zz) [reply src/common/config/index.tsx]: done

File: src/common/config/index.tsx (hunk 3 above)
Comment (Nandan-unni-zz) [reply src/common/config/index.tsx]: done

**Pull request merged**